api/services/rdProfessionals.ts

```diff
@@ -4,7 +4,7 @@ import {http} from '../lib/http';
 export async function getOrganisationId(details) {
   // TODO remove the hardcoded email when correct user gets returned from idam
   // const email = details.data.email;
-  const email = 'henry_fr_harper@yahoo.com'
-  return await http.get(`${config.services.rdProfessionalApi}/search/organisations/${email}`)
+  const email = 'juitestuser2@gmail.com'
+  return await http.get(`${config.services.rd_professional_api}/search/organisations/${email}`)
```

Comment:
please revert as per alan we are using camelCase rather the under_score

Reply:
uu missed this one